decidim-participatory_processes/app/queries/decidim/participatory_processes/filtered_participatory_process_groups.rb

```diff
@@ -18,6 +18,8 @@ def query
                       processes.where("decidim_participatory_processes.end_date <= ?", Date.current)
```

Comment:
I am just looking at this snippet, and i wonder if we could use the scopes defined on the participatory_process...

https://github.com/decidim/decidim/blob/f1ae813476b6d0d172f86f41a93ca8cef184b249/decidim-participatory_processes/app/models/decidim/participatory_process.rb#L77-L79

Reply:
That's a great idea. Didn't check the scopes but this is now fixed by 89bbacea749255a83fbcf88d6e5eb3e559cd579f.